study.md

```diff
@@ -147,5 +162,5 @@ using `hank` that equals "My name is Hank Hill and I'm a propane and propane
 accesories salesman that enjoys drinking beer.".
 
 ```javascript
-// your answer here
+`My name is ${hank['givenName']} ${hank['familyName']} and I'm a ${hank['hobbies'][1]} salesman that enjoys ${hank['hobbies'][0]}`;
```

Comment:
This prints correctly but I was expecting you to use `hank.occupation` rather than `hank.hobbies[1]` with " salesman" after it.